yt/frontends/ramses/data_structures.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, ds, domain_id):
 
         # Autodetect particle files
         particle_handlers = [
-            PH(ds, self) for PH in get_particle_handlers() if PH.any_exist(ds)
+            PH(self) for PH in get_particle_handlers() if PH.any_exist(ds)
```

Comment:
I went back and forth with this line and I don't understand how both the old and the new versions can be correct.